tools/RAiDER/delay.py

```diff
@@ -229,9 +228,8 @@ def _build_cube_ray(
     epsg4326 = CRS.from_epsg(4326)
     cube_to_llh = Transformer.from_crs(pts_crs, epsg4326,
                                        always_xy=True)
-    ecef_to_model = Transformer.from_crs(CRS.from_epsg(4978), model_crs)
-    # For calling the interpolators
-    flip_xy = model_crs.axis_info[0].direction == "east"
```

Comment:
@piyushrpt @jlmaurer and I couldn't find any use for this... do you remember why it was here, perhaps for NCMR?